jar-infer-lib/README.md

```diff
@@ -22,5 +22,11 @@ This will pull in the required WALA jars and build the analysis code.
 
 ### Usage
```

Comment:
Shouldn't this be in a `jar-infer-cli/README.md` rather than on the one for `jar-infer-lib`?